### ISSUE TITLE
Outlook 2010- Message edit is announced on reaching the uneditable message body of a recieved message when message is opened in a separate window.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3019]
**Reported by sumandogra on 2013-02-22 14:46**
Outlook 2010
NVDA announces message edit on reaching the message body in a received message of any form either a plain text or HTML form.  This is confusing as there is not editable field there.

Steps to reproduce:
1.  Open a received message in Outlook 2010 with ENTER.
2.  The message opens in a new window.
3.  NVDA automatically starts reading the info.
4.  On reaching the message body, NVDA announces message edit.
5.  This does lead to confusion at times, as it is not an editable field.

**Comment by leonardder** [comment]
@sumandogra: Can you still reproduce this?

Note that this might be a real editable field, but it is probably read only. Does NVDA provide the latter information as well?

**Comment by Adriani90** [comment]
@leonardder I cannot reproduce this in MS Outlook 2016. NVDA reads "document multi line". Nothing about editable field or so.
I can test tomorrow in Outlook 2010.

**Comment by Adriani90** [comment]
@leonardder I cannot reproduce it in Outlook 2010 either.

**Comment by ehollig** [comment]
Thank you @Adriani90 for your extensive testing. Closing
